src/test/ui/did_you_mean/issue-40396.stderr

```diff
@@ -4,31 +4,31 @@ error: chained comparison operators require parentheses
 12 |     println!("{:?}", (0..13).collect<Vec<i32>>());
    |                                     ^^^^^^^^
    |
-   = help: use `::<...>` instead of `<...>` if you meant to specify type arguments
+   = help: use `::<...>` instead of `<...>` if you meant to specify type arguments or use `(...)` if you meant to specify fn arguments
```

Comment:
This will probably look better as a second `help:` on a separate line.

Reply:
I think I also would prefer a second help